test/unit/host_test.rb

```diff
@@ -416,4 +416,13 @@
 
     end
   end
+
+
+  context "DisassociateCommand" do
```

Comment:
Would you mind adding also a functional test for the list command? We prefer covering commands with functional tests because they provide much more readable errors, test the command in the context of the whole cli and uses real data (compared to default responses from apidoc).
You can see an example here: https://github.com/theforeman/hammer-cli-foreman/blob/master/test/functional/personal_access_token_test.rb#L55-L80